### PR TITLE
Run fmt on all files

### PR DESCRIPTION
**Pull request by sids-b**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Run fmt on all files

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by fisherxu** [comment]
/lgtm
cc @daixiang0 Is the files under staging not checked?

**Comment by daixiang0** [comment]
@fisherxu @kevin-wangzefeng In fact, those issues belong to other linters not "gofmt" :(

File: staging/src/github.com/kubeedge/beehive/pkg/core/context/context_factory.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ func InitContext(contextType string) {
 	})
 }
 
-func GetContext() gocontext.Context{
+func GetContext() gocontext.Context {
```
Comment (fisherxu) [inline staging/src/github.com/kubeedge/beehive/pkg/core/context/context_factory.go:43]: I think this line is gofmt issue? @daixiang0

Comment (daixiang0) [reply staging/src/github.com/kubeedge/beehive/pkg/core/context/context_factory.go:43]: The gofmt linter only lint files from `go list ./...`，unlucky, the `staging` dir is not in this list.
Use `gofmt -l -w ./..` can check `vendor` and `staging` dir.

Comment (fisherxu) [reply staging/src/github.com/kubeedge/beehive/pkg/core/context/context_factory.go:43]: Vendor should not in our check scope..

**Pull request merged**